tests/ctrl/test_sim_devices.py

```diff
@@ -0,0 +1,52 @@
+import time
+import pytest
```

Comment:
import は空行を使って [標準ライブラリ](https://docs.python.org/ja/3/library/)、外部パッケージ、[同一パッケージ内](https://docs.python.org/ja/3/reference/import.html#package-relative-imports) の3グループに分けてほしいです。

```suggestion
import time

import pytest
```